alerts/machine-config-operator/MachineConfigDaemonPivotError.md

```diff
@@ -0,0 +1,146 @@
+# MCDPivotError
+
+## Meaning
```

Comment:
I wonder if there is a way to combine these two paragraphs into a single, concise one as they're both describing close to the same thing. Just a thought, not blocking.

Reply:
You mean these two paragrapths?

~~~
This alert is triggered when the Machine Config Daemon (MCD)
detects an error when trying to pivot the
operating system (OS) image to a new version
or is unable to pivot to a new version during a cluster upgrade.

This alert occurs when a node experiences
OS update or kernel change failure(s)
lasting more than 2 minutes during a cluster upgrade.
~~~

Reply:
Okay cut this down to 1 paragraph then.